the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+        case 3:
+            cell.textLabel?.text = lastSectionLabels[indexPath.row]
+            
+            cell.textLabel?.textColor = .red
+            cell.textLabel?.textAlignment = .center
+            
+        default:
+            fatalError("This row does not exist")
+        }
+        
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.section {
+        case 0:
+            // Handle authentification/unauthentification stuff
+            break
+            
+        case 1:
+            break
+            
+        case 2:
+            var urlString: String?
+            
+            switch indexPath.row {
+            case 0:
+                urlString = "https://www.thebluealliance.com"
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 1:
+                urlString = "https://github.com/the-blue-alliance/the-blue-alliance-ios"
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 2:
+                // TODO: Make a contributors list on Github
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 3:
+                // TODO: Make a changelog
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            default:
+                fatalError("This row does not exist")
+            }
+            
+            if let urlString = urlString {
+                let url = URL(string: urlString)
+                if let url = url, UIApplication.shared.canOpenURL(url) {
+                    UIApplication.shared.open(url, options: [:], completionHandler: nil)
+                }
+            }
+            
+        case 3:
+            if indexPath.row == 0 {
+                let alertController = UIAlertController(title: "Delete network cache", message: "Are you sure you want to delete all the network cache data ? This action is irreversible.", preferredStyle: .alert)
+                
+                let confirmAction = UIAlertAction(title: "Delete", style: .destructive) { (deleteAction) in
+                    // TODO: Handle network cache deletion
+                }
+                
+                let cancelAction = UIAlertAction(title: "Cancel", style: .cancel) { (deleteAction) in
+                    self.tableView.deselectRow(at: indexPath, animated: true)
+                }
+                
+                alertController.addAction(confirmAction)
+                alertController.addAction(cancelAction)
+                
+                self.present(alertController, animated: true, completion: nil)
+            } else {
+                let alertController = UIAlertController(title: "Delete app data", message: "Are you sure you want to delete the app data ? This action is irreversible.", preferredStyle: .alert)
```

Comment:
Let's remove the space after the question mark on these. Also - this message seems needlessly scary. Let's just keep the first line of this. I'd like to frame this action as "Deleting the app data is a good idea if your app gets in to a weird state, and don't worry, we can always redownload all the data" but that's not good copy haha